### PR TITLE
Remove duplicate share badges

### PR DESCRIPTION
**Pull request by maxxcrawford**
Check the following sites: 

To test – You should only see ONE share badge per page. On any other FB share buttons, hover over button to make that badge appear. 

+ https://www.estadao.com.br/
+ https://www.theguardian.com/commentisfree/2019/may/24/next-tory-leader-panel-theresa-may-conservatives#comments
+ https://www.buzzfeed.com/bekoconnell/cheap-products-everything-50

### CODE REVIEW

**Comment by maxxcrawford** [comment]
@arthur-mozilla See the following: 
[facebook_container-1.9.2.zip](https://github.com/mozilla/contain-facebook/files/3229714/facebook_container-1.9.2.zip)

**Comment by maxxcrawford** [comment]
_Note: The 1.9.2 Zip is stale as of the last commit._

Resolved Guardian index issue: https://www.theguardian.com/world/2019/may/27/abortion-democrats-republicans-state-laws

image

File: src/content_script.js [hunk 1]
```diff
@@ -325,7 +348,7 @@ function calcZindex(target) {
   }
 
   // Take highest zindex in parent tree and adds one more.
-  zIndexLevel = zIndexLevel + 1;
+  zIndexLevel = zIndexLevel + 2;
```
Comment (groovecoder) [inline src/content_script.js]: Do we really need just this 1 additional level?

**Comment by groovecoder** [comment]
Feature is looking good.

**Pull request merged**